topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -67,6 +71,7 @@ async def load_metadata(self, asset: Asset) -> None:
                 "linz:photo_version": asset_metadata["photo_version"],
             }
         )
+        item.add_extension("https://linz.github.io/stac/v0.0.2/camera/schema.json")
```

Comment:
Did we decide on if `camera_sequence_no` and `nominal_focal_length` are both null do we need a `camera` schema added?

Reply:
The camera extension currently doesn't allow for null values, so I've upadated the code to only add the fields and extension if the values exist.